### ISSUE TITLE
Lecture - perspective - what do I need for learning to code

### ISSUE THREAD

**Issue by EggDice** [talk2machine/talk2machine#16]
- [ ] Script
- [ ] Proof read
- [ ] Search for assets
- [ ] Screen record
- [ ] Record
- [ ] Check for all clips
- [ ] Sound
- [ ] Cut
- [ ] Edit
- [ ] Proof watch

Scratch
 - What do I need for learning to code?
 - The most important is to understand English
   Unfortunately, the best materials for learning programming and helpful websites are in English
 - Level?
 - If you understand this you are fine
    in my videos, I help you to understand the terms
 - But I bet I need an expensive computer
 - It is a lie that coders are telling to their employers, and themselves, because they would like to have cool computers
 - Cool computers cut
 - There are only a few fields where you need a beefy machine, and in most cases, you can solve that with some cloud provider.
 - Cloud provider?
 - Basically, companies own a huge amount of computers and they rent them to use them through the internet
 - So what should I look for in a machine if I want to buy one
 - The most important is the ram it should be at least 8GB, but basically, if you buy this today for $0000 then you are good to go
   But if you bought it 8 years ago for the same maybe you need a new one in the long term
 - old computer cut
 - What else do I need?
 - Beside your motivation it is just a browser for starting (preferably chrome)
